### PR TITLE
new request

### PR DESCRIPTION
**Pull request by AndreyStrogush**

### CODE REVIEW

File: src/carExample/Car.java [hunk 1]
```diff
@@ -15,12 +16,12 @@
     private List<CarDoors> doors = new ArrayList<>();
 
     public Car(String productionDate) {
```
Comment (Taniann) [inline src/carExample/Car.java]: why do you pass String productionDate change it to LocalDate as you've changed above

File: src/carExample/Car.java [hunk 2]
```diff
@@ -15,12 +16,12 @@
...
-        this.productionDate = productionDate;
+        this.productionDate = LocalDate.now();
```
Comment (Taniann) [inline src/carExample/Car.java]: and use there the passed parameter then

File: src/carExample/Car.java [hunk 3]
```diff
@@ -15,12 +16,12 @@
...
     }
 
     public Car(String productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
```
Comment (Taniann) [inline src/carExample/Car.java]: same as above: String productionDate

**Review by Taniann — CHANGES_REQUESTED** [review]

File: src/carExample/Car.java (hunk 3 above)
Comment (AndreyStrogush) [reply src/carExample/Car.java]: just inattention, sorry
already push it to master

File: src/carExample/Car.java [hunk 4]
```diff
@@ -14,13 +15,13 @@
...
-        this.productionDate = productionDate;
+    public Car(LocalDate productionDate) {
+        this.productionDate = LocalDate.now();
     }
 
-    public Car(String productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
+    public Car(LocalDate productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
                int maxNumberOfPassengers, int actualNumberOfPassengers, int currentSpeed) {
-        this.productionDate = productionDate;
+        this.productionDate = LocalDate.now();
```
Comment (Taniann) [inline src/carExample/Car.java]: you passed productionDate as constructor param but dont use it in constructor body

**Review by Taniann — CHANGES_REQUESTED** [review]

File: src/carExample/Car.java (hunk 4 above)
Comment (AndreyStrogush) [reply src/carExample/Car.java]: added into main method

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**